services/alertmanager/alertmanager.go

```diff
@@ -73,12 +81,50 @@ func (svc *Service) Run(ctx context.Context) {
 	}
 }
 
+// createDataDir creates alert manager directories if not exists in the persistent volume.
+func (svc *Service) createDataDir() {
+	group, err := user.Lookup("pmm")
```

Comment:
Nope. We should stat `/srv/prometheus` and use the same uid and gid. Remember, that file is on the volume.

Reply:
Added.